examples/epstein_civil_violence/epstein_civil_violence/model.py

```diff
@@ -68,9 +70,9 @@ def __init__(
             "Cops": self.count_cops,
         }
         agent_reporters = {
-            "x": lambda a: a.pos[0],
-            "y": lambda a: a.pos[1],
-            "breed": lambda a: a.breed,
+            "x": lambda a: a.cell.coordinate[0],
+            "y": lambda a: a.cell.coordinate[1],
```

Comment:
Can we make properties for this? Like `cell.x`, `cell.y` and `cell.z`? Similar like https://github.com/projectmesa/mesa/pull/2348 but for the cell.